src/components/images/imageLoader.js

```diff
@@ -132,15 +114,21 @@ import 'css!./style';
         }
         elem.setAttribute('data-src', url);
 
-        if (elem.classList.contains('non-blurhashable') || !userSettings.enableBlurhash()) {
-            elem.classList.remove('lazy-image-fadein-fast', 'lazy-image-fadein');
-            elem.classList.add('lazy-hidden');
-        } else {
-            switchCanvas(elem);
-        }
+        elem.classList.remove('lazy-image-fadein-fast', 'lazy-image-fadein');
+        elem.classList.add('lazy-hidden');
     }
 
     export function lazyChildren(elem) {
+        if (userSettings.enableBlurhash()) {
+            for (const lazyElem of elem.getElementsByClassName('lazy')) {
+                const blurhashstr = lazyElem.getAttribute('data-blurhash');
```

Comment:
just `blurhash` seems fine

Reply:
@dkanada I think this might lead into confusion, as we also have the module Blurhash (that we use above for decoding)

Reply:
It should at least use camel casing then.

Reply:
@dkanada Fair enough :)